### PR TITLE
Find tests on windows

### PR DESCRIPTION
**Pull request by markus-ferrell**
Regression test is still marked with skip because install does not pass unit tests.

### CODE REVIEW

**Comment by johnwparent** [comment]
@spackbot fix style

**Comment by johnwparent** [comment]
@scheibelp If this PR passes CI, it's ready for a merge/review/what have you.

**Review by johnwparent — DISMISSED** [review]
Pending CI success.

File: var/spack/repos/builtin.mock/packages/dyninst/package.py [hunk 1]
```diff
@@ -30,6 +30,5 @@ class Dyninst(Package):
     depends_on("libdwarf")
 
     def install(self, spec, prefix):
-        configure("--prefix=%s" % prefix)
-        make()
-        make("install")
+        mkdirp(prefix)
```
Comment (johnwparent) [inline var/spack/repos/builtin.mock/packages/dyninst/package.py:33]: Something here is causing CI failures on MacOS and Linux

**Comment by markus-ferrell** [comment]
@spackbot fix style

**Review by johnwparent — APPROVED** [review]

**Comment by johnwparent** [comment]
Looks good! @scheibelp can you give this a once over and then LGTM?

**Review by scheibelp — APPROVED** [review]

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**